ui/main/src/app/business/buildInTemplates/message-or-question-list/usercard/message-or-question-listUserCardTemplateView.ts

```diff
@@ -0,0 +1,88 @@
+/* Copyright (c) 2023, RTE (http://www.rte-france.com)
+ * See AUTHORS.txt
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ * SPDX-License-Identifier: MPL-2.0
+ * This file is part of the OperatorFabric project.
+ */
+
+declare const opfab;
+declare const usercardTemplateGateway;
+
+export class MessageOrQuestionListUserCardTemplateView {
+
+    
+    messageOrQuestionList;
+    selectedMessage;
+
+    public getMessage() {
+        let message = opfab.currentCard.getCard()?.data?.message;
+        if (message) message = opfab.utils.convertSpacesAndNewLinesInHTML(opfab.utils.escapeHtml(message));
+        else message = '';
+        return message;
+    }
+
+
+    public getSpecificCardInformation(message: string) {
+        message = message.trim();
+        if (message.length === 0) {
+            return {
+                valid:false,
+                errorMsg : opfab.utils.getTranslation('buildInTemplate.message-or-question-listUserCard.emptyMessage') 
+            }
+        }
+
+        let title = this.selectedMessage.title.trim();
+        let question = this.selectedMessage.question;
+        let severity = 'INFORMATION'
+        if (this.selectedMessage.question) {
+            severity = 'ACTION'
+        }
+
+        const card = {
+		title : {key : "message_or_question_list.title", parameters : {"messageTitle" : title} },
+            summary: {key : "message_or_question_list.summary"},
+            startDate: new Date().valueOf(),
+            data: {
+                title: title,
+                message: message,
+                question: question
+            },
+            severity: severity
+        };
+
```

Comment:
add field entitiesAllowedToRespond with the list of recipients when it is a question otherwise set value to ""